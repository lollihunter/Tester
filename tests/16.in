20 20 20
1 1
3 4
13 6
11 18
10 20
3 16
3 19
14 17
2 1
8 10
15 17
11 16
15 3
12 3
12 12
15 2
15 10
8 4
19 14
20 20
