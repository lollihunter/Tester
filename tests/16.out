9
1 2 18 14 3 17 19 11 20 
